### PR TITLE
feat(a380x/flex): Wing flex tuning 

### PR DESCRIPTION
**Pull request by crocket63**
Fixes #[issue_no]

## Summary of Changes

Trying to add a bit more flexibility while keeping flex under control at min or max fuel load.

## Additional context

Discord username (if different from GitHub):

## Testing instructions

This is mainly a "feeling check" and trying to cross check with any YouTube reference:

Check against any findable reference that full fuel load gives correct wing flex on ground. Same for empty of fuel.
Check against any findable reference that wing bend looks correct from same window views as real life references
Check that wing flex do not go crazy on hard landings.
Check that smooth landing gives correct flex oscillations
Check that wing drops with ground spoiler deployed 

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 1. [A380X/MFD] Fixed DIRECT TO selection of active waypoint does nothing - @sognodelx (Sven Gross)
 1. [A380X/FLIGHT MODEL] Fix for stall and auto-rotation issues- @donstim (donbikes)
 1. [A380X/MODEL] Optimized 4K textures - @Repsol
+1. [A380/WingFlex] Reduced stiffness of wings for more tip up bend - @Crocket63 (crocket)
```
Comment (2hwk) [inline .github/CHANGELOG.md:32]:
```suggestion
1. [A380/WING_FLEX] Reduced stiffness of wings for more tip up bend - @Crocket63 (crocket)
```

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: .github/CHANGELOG.md (hunk 1 above)
Comment (2hwk) [reply .github/CHANGELOG.md:32]: Just a minor consistency change for changelog, no code change affected.

**Comment by AlonsoJCY** [comment]
I can give a evidence for you to check. Hope it is helpful :)
Comp wing flex
LHS is simulator; RHS is IRL.
LHS from PVG to PEK; RHS from PVG to FRA
Even if the fuel load on the RHS is greater than on the LHS, RHS still has more wing flex.

**Comment by A359Flex** [comment]
Any way to modify or change this tuning as per the user's desire in case if he needs to do it by himself?

**Comment by A359Flex** [comment]
> > Any way to modify or change this tuning as per the user's desire in case if he needs to do it by himself?
> 
> Well flex is not user desire based, it's physics... So answer is: if one find references with good amount of information that can give insights to improve the tuning, then anyone can propose a new tuning in here, it's open source.
> 
> But having a wing flex setting in EFB would be non sense to me.
> 
> Also lot of possible improvement in the dynamic would need Asobo changes. Feel free to upvote this request: https://devsupport.flightsimulator.com/t/can-we-access-surfaces-lift-values/5574?u=davy63000

Fine, but what I've noticed was that despite increasing gross weights and during conditions of high pitch attitudes, the wing remained relatively stiff which isn't anything like it behaves in real life where the deflection of the outer 2/3rds does vary with the above-mentioned factors. 
Microsoft Flight Simulator Screenshot 2024 10 30 - 14 18 11 79
A380-Emirates-Golf-Oscar-Romeo
shutterstock_112881910-1

**Comment by A359Flex** [comment]
> > > > Any way to modify or change this tuning as per the user's desire in case if he needs to do it by himself?
> > 
> > 
> > > 
> > 
> > 
> > > Well flex is not user desire based, it's physics... So answer is: if one find references with good amount of information that can give insights to improve the tuning, then anyone can propose a new tuning in here, it's open source.
> > 
> > 
> > > 
> > 
> > 
> > > But having a wing flex setting in EFB would be non sense to me.
> > 
> > 
> > > 
> > 
> > 
> > > Also lot of possible improvement in the dynamic would need Asobo changes. Feel free to upvote this request: https://devsupport.flightsimulator.com/t/can-we-access-surfaces-lift-values/5574?u=davy63000
> > 
> > 
> > Fine, but what I've noticed was that despite increasing gross weights and during conditions of high pitch attitudes, the wing remained relatively stiff which isn't anything like it behaves in real life where the deflection of the outer 2/3rds does vary with the above-mentioned factors.
> > Microsoft Flight Simulator Screenshot 2024 10 30 - 14 18 11 79
> > A380-Emirates-Golf-Oscar-Romeo
> > shutterstock_112881910-1
> 
> For example you don't have same angle at all on the first two so can't compare anything. Other example, this is without the tuning in this PR: image

It is impossible to get the two very exact same angles in space and time so that argument is moot as long as there is closeness and even in your screenshot the outboard wing shows slightly more deflection. And there's plenty of evidence that the FBW A380X right now does not exhibit sufficient flexing behaviour as seen in the real aircraft.
382172406-a3e576a3-f2f7-41b9-ae50-633f4e3b5591
Microsoft Flight Simulator Screenshot 2024 11 02 - 08 13 12 100
382957844-b5bebfe3-2baf-4416-b658-0800bc8e257a

**Comment by Benjozork** [comment]
There are so many pictures and quotes of pictures that it's getting hard to see messages strictly related to the PR. I suggest you continue this conversation on Discord.

**Review by 2hwk — APPROVED** [review]

**Comment by alepouna** [comment]
Reminder for observers of this PR, that for flex comparisons with our aircraft and the IRL a380, we need additional data such as the weight, fuel and the "POVs" to match in both comparisons, otherwise it isn't really helpful!

**Comment by beheh** [comment]
I think the "Needs Accuracy Check" label here is mostly moot. It makes sense for systems that can be looked up or tested and compared 1:1, but wing flex, like graphical choices such as textures or lighting, are always going to be somewhat subjective.

**Comment by 2hwk** [comment]
Quality Assurance Report

Discord Username: 2cas
Object of testing: #9424
Aircraft: A380X 
Tier of Testing: 1
Date: 10/11/2024 

Testing Process: 
 
- [x] Check against any findable reference that full fuel load gives correct wing flex on ground. Same for empty of fuel.
   - Compared to gate photos of known routes and fuel levels
- [x] Check against any findable reference that wing bend looks correct from same window views as real life references
   - Compared to videos of known fuel levels, historical conditions, etc.
- [x] Check that wing flex do not go crazy on hard landings.
  - Slammed it and bounced on purpose, used slew mode to throw the plane into the runway. Wing flex does not break.
- [x] Check that wing drops with ground spoiler deployed

Testing Results: 
QA Passed  

Negatives:       

-  Same issue as before during takeoff rollout (due to not having the sim flight model calculation's actual wing lift values.
  - i.e. wing does not lift before rotation, then snaps into place, that limitation still exists.

Conclusions: 

- There is more perceived wing flex, still the same issue as before during takeoff rollout (due to not having the sim flight model calculation's actual wing lift values.

**Comment by 2hwk** [comment]
> > Quality Assurance Report
> > Discord Username: 2cas Object of testing: #9424 Aircraft: A380X Tier of Testing: 1 Date: 10/11/2024
> > Testing Process:
> > 
> > * [x]   Check against any findable reference that full fuel load gives correct wing flex on ground. Same for empty of fuel.
> >   
> >   * Compared to gate photos of known routes and fuel levels
> > * [x]   Check against any findable reference that wing bend looks correct from same window views as real life references
> >   
> >   * Compared to videos of known fuel levels, historical conditions, etc.
> > * [x]   Check that wing flex do not go crazy on hard landings.
> >   
> >   * Slammed it and bounced on purpose, used slew mode to throw the plane into the runway. Wing flex does not break.
> > * [x]   Check that wing drops with ground spoiler deployed
> > 
> > Testing Results: QA Passed
> > Negatives:
> > 
> > * Same issue as before during takeoff rollout (due to not having the sim flight model calculation's actual wing lift values.
> > * i.e. wing does not lift before rotation, then snaps into place, that limitation still exists.
> > 
> > Conclusions:
> > 
> > * There is more perceived wing flex, still the same issue as before during takeoff rollout (due to not having the sim flight model calculation's actual wing lift values.
> 
> Mmmmm I'm now blending ground and air mode slowly so it should to some extent raise during the to roll even if last bit happens at lift off ? At rotation you also get AoA so it's expected to have a bit of flex punch there though.

There was some rise towards the end of the TO roll but it does seem slightly delayed. Good enough to merge into master as it is better than before.

**Pull request merged**